servers/physics/body_pair_sw.cpp

```diff
@@ -46,6 +46,7 @@
 //#define ALLOWED_PENETRATION 0.01
 #define RELAXATION_TIMESTEPS 3
 #define MIN_VELOCITY 0.0001
+#define MAX_BIAS_ROTATION (Math_PI/8)
```

Comment:
We may want to expose this in the future. I've tested with this value and it seems to work well in a few cases